### ISSUE TITLE
calligra fails to build after the last revbump.

### ISSUE THREAD

**Issue by fbrosson** [haikuports/haikuports#2701]
In #2700 I pushed several commits, one of them being a `REVISION` bump in 8a29de83ef458a9d868da0f80101e160cf765ee5 for calligra, with a few cosmetic edits and whitespace fixes.
Unfortunately, the build failed:
http://kitchen.haiku-os.org/master/x86_64/buildruns/1569/builds/10929.log
http://kitchen.haiku-os.org/master/x86_gcc2/buildruns/1411/builds/28278.log
The build fails on both architectures with similar messages:
```
[ 20%] Building CXX object libs/flake/CMakeFiles/flake.dir/flake_autogen/mocs_compilation.cpp.o
[ 20%] Linking CXX shared library libflake.so
[ 20%] Built target flake
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
Warning: Command '['bash', '-c', '. /wrapper-script']' returned non-zero exit status 2
Error: Build has failed - stopping.
```
Unlike the build for gsl itself, which I had tested locally on all available architectures, I did not kick a build for calligra. It was a blind merge. (I'll avoid doing blind commits in the future.)

**Comment by diversys** [comment]
Fix build with Qt 5.11 (missing headers) https://phabricator.kde.org/D11454

**Comment by threedeyes** [comment]
Fixed in 167dc67144fa0cb030bc06533a6fc217373a5e45
